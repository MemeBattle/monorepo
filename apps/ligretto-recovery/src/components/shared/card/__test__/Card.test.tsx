import React from 'react'
import renderer from 'react-test-renderer'

import { Card } from '../index'

describe(Card, () => {
  it('Should render', () => {
    const component = renderer.create(<Card />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
