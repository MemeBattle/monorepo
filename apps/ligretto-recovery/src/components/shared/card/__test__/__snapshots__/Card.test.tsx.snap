// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should render 1`] = `
<div
  className="card empty"
  onClick={[Function]}
>
  <div
    className="value"
  />
</div>
`;
